gui/qt/util.py

```diff
@@ -63,6 +63,8 @@ def __init__(self, text, func):
     def keyPressEvent(self, e):
         if e.key() == Qt.Key_Return:
             self.func()
+        else:
+            e.ignore()
```

Comment:
What the fuck github ate my comments.   I wrote a bunch of stuff here.

I basically said this: From Qt docs: https://doc.qt.io/qt-5/qwidget.html#keyPressEvent  

> This event handler, for event event, can be reimplemented in a subclass to receive key press events for the widget.
> A widget must call setFocusPolicy() to accept focus initially and have focus in order to receive a key press event.
> If you reimplement this handler, it is very important that you call the base class implementation if you do not act upon the key.

Basically you want to do `QPushButton.keyPressEvent(e)` here.  The bug is because Electrum programmers don't read docs.  That should fix it. And is future-proofing the code.

Reply:
OK, I'm dropping this commit for now since it's minor, and we should probably do a more comprehensive fix everywhere (this keyPressEvent thing gets overridden in many more places)